README.md

```diff
@@ -70,6 +71,15 @@ store.dispatch({
 });
 ```
 
+When you want to force the finish of all pending tasks (for example: with a global error).
```

Comment:
Could you also add that it is important to make sure you don't have any pending tasks which might com back and trigger an `end` which will cause the counter to drop below 0.

Reply:
I already added the tests for my changes.
Regarding the update in the documentation, I think it is not necessary because having a new `actionKey` (`endAll`) for this specific situation, the counter will never drop below 0, since the idea is to always set it to 0.

Reply:
My reasoning was that you may have an outstanding request, which did ` [ pendingTask ]: start`, then for some reason you do `endAll`, then the outstanding request finished, where you dispatch `[ pendingTask ]: end` - at this point you've set it to 0 with `endAll` and the `[ pendingTasl ]: end` will set it to -1.

This is of course a developer error, not a problem with your implementation. I'll update the docs if there are frequent issues with it.